server/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+server.servlet.context-path: /travelista/api
+
+spring:
+  profiles:
+    include: destinations
+
+#api
+api:
+  google:
+    googleApiKey: ${GOOGLE_API_KEY}
```

Comment:
this will be environment variable required for server application to start. Do you think is good naming? Or should i rename to something else like geolocation_api_key (with uppercase)?

Reply:
I think this will suffice.